godot-core/src/builtin/math.rs

```diff
@@ -222,11 +237,18 @@ mod test {
     #[test]
     fn lerp_angle_test() {
         assert_eq_approx!(lerp_angle(0.0, PI, 0.5), -FRAC_PI_2, is_angle_equal_approx);
+        #[cfg(not(feature = "double-precision"))]
         assert_eq_approx!(
             lerp_angle(0.0, PI + 3.0 * TAU, 0.5),
             FRAC_PI_2,
             is_angle_equal_approx
         );
+        #[cfg(feature = "double-precision")]
+        assert_eq_approx!(
+            lerp_angle(0.0, PI + 3.0 * TAU, 0.5),
+            -FRAC_PI_2,
+            is_angle_equal_approx
+        );
```

Comment:
Why is this `pi / 2` for floats, but `-pi / 2` for doubles?
Rounding errors?

Could you please add a TODO so we write a more robust angle comparison function later (outside of scope for this PR, let's not delay further 🙂)?